dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
-pyyaml
```

Comment:
Did you mean to remove it here and add it somewhere else or is removing it a typo?

Reply:
I unintentionally deleted this when I added the comment, fixing it now